# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBCursor.cpp
# Commit: d7c936098f29
# Full Hash: d7c936098f29599d4d1b4ef2519de3b8a43948e5
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-03-31 22:58:23
# Description:
#   Bug 1625444 - Make IDBObjectStoreCursor a multi-zone JSHolder class. r=jonco,dom-workers-and-storage-reviewers,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D68766
# ==============================================================================

diff -r 13865519a6e9 -r d7c936098f29 dom/indexedDB/IDBCursor.cpp
--- a/dom/indexedDB/IDBCursor.cpp	Tue Mar 31 08:21:39 2020 +0000
+++ b/dom/indexedDB/IDBCursor.cpp	Mon Mar 30 18:31:04 2020 +0000
@@ -837,7 +837,8 @@
   NS_IMPL_CYCLE_COLLECTION_MULTI_ZONE_JSHOLDER_CLASS(_subclassName)      \
   NS_IMPL_CYCLE_COLLECTION_IDBCURSOR_SUBCLASS_METHODS(_subclassName)
 
-NS_IMPL_CYCLE_COLLECTION_IDBCURSOR_SUBCLASS(IDBObjectStoreCursor)
+NS_IMPL_CYCLE_COLLECTION_IDBCURSOR_MULTI_ZONE_JSHOLDER_SUBCLASS(
+    IDBObjectStoreCursor)
 NS_IMPL_CYCLE_COLLECTION_IDBCURSOR_SUBCLASS(IDBObjectStoreKeyCursor)
 NS_IMPL_CYCLE_COLLECTION_IDBCURSOR_MULTI_ZONE_JSHOLDER_SUBCLASS(IDBIndexCursor)
 NS_IMPL_CYCLE_COLLECTION_IDBCURSOR_SUBCLASS(IDBIndexKeyCursor)
